DEVELOP.md

```diff
@@ -62,7 +62,7 @@ specified:
 as follows:
 
 ```console
-$ go get k8s.io/client-go@v8.0.0
+$ go get k8s.io/client-go@kubernetes-1.15.2
 $ go get k8s.io/api@release-1.11
```

Comment:
IIRC, this and `k8s.io/apimachinery` need not be updated manually
since `k8s.io/client-go` contains appropriate `go.mod`.